src/shared/components/palette/address/address-option.model.ts

```diff
@@ -7,11 +7,12 @@ export class AddressOption {
   constructor (addressModel: AddressModel, description: string) {
     this.value = addressModel;
     this.description = (description == null)
-        ? this.value.AddressLine1
+        ? (this.value.AddressLine1
             +  this.prefixWithCommaIfPresent(this.value.AddressLine2)
```

Comment:
minor: perhaps better extract this block into a private method?

Reply:
👍